db/migrate/20150317210343_convert_target_minutes_to_hours.rb

```diff
@@ -0,0 +1,13 @@
+class ConvertTargetMinutesToHours < ActiveRecord::Migration
+  class TempTarget < ActiveRecord::Base
+    self.table_name = 'targets'
+  end
```

Comment:
@ChrisBr thanks for the note. That was a new thing to learn.

I am now wondering if we would need a ```down``` method for rollback changes, where would need to shift the hours back to minutes? Pretty much reverse whatever and exactly how it is happening in the ```up``` method?